presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestTypedSet.java

```diff
@@ -130,9 +134,29 @@ public void testBigintSimpleTypedSet()
         }
     }
 
+    @Test
+    public void testMemoryExceeded()
+            throws Exception
+    {
+        try {
+            TypedSet typedSet = new TypedSet(BIGINT, 10, FUNCTION_NAME);
+
+            for (int i = 0; i <= TypedSet.FOUR_MEGABYTES + 1; i++) {
+                Block block = createLongsBlock(Collections.nCopies((int) 1, (long) i));
+
+                typedSet.add(block, 0);
+            }
+
+            fail("expected exception");
+        }
+        catch (PrestoException e) {
+            assertEquals(e.getErrorCode(), StandardErrorCode.EXCEEDED_FUNCTION_MEMORY_LIMIT.toErrorCode());
```

Comment:
i'd like to talk with you offline about this at some at some point. I've left this in as I've preferred the extra context of seeing what class the enum/constant/method is from right there. in the code. is this thinking that intellij can provide that context by cmd-hovering?
or is it some methods / constant in contexts are not ambiguous?

example is MultimapAggregationFunction.NAME inside MultiKeyValuePairs. it makes a lot of sense since "NAME" is so vague. again, i guess you could argue you could cmd-hover/click to see where name lives and its value; but the className.VAR kind of says a lot imo

let me know if you want htis one imported too (you hadn't mentioned yet)